HHS/Head-Start-TTADP
# [Pord] Monitoring: getting stuck trying to re-download the same file repeatedly

## Description of change

- If a file has failed to download 5 times, move on to the next file.
- Don't try to download a file that has already been processed.
- TTAHUB-1095: Limit email preferences based on user role.


## How to test
Successfully download and process a file,  when a new file is added the to SFTP confirm it does not try to process the same file again.
When a file has failed multiple times, confirm it moves on to the next identified file.

TTAHUB-1095: Make sure you can set e-mail preferences for users with appropriate roles.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2679
* https://jira.acf.gov/browse/TTAHUB-1095


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
